site/package-lock.json

```diff
@@ -3491,13 +3491,15 @@
           "version": "1.0.0",
```

Comment:
You should checkout this file. Here's the related npm issue: https://github.com/npm/npm/issues/17722

Reply:
This file still needs to be checkout.